src/containeirs/NoMatch/styled.ts

```diff
@@ -0,0 +1,115 @@
+import styled, { css, keyframes } from 'styled-components';
+import { ReactComponent as Logo } from 'assets/logo.svg';
+import { Button } from 'components/Button';
+
+const pulse = keyframes`
+    0% {
+     transform: scale(1);
+    }
+    50% {
+     transform: scale(1.1);
+    }
+    100% {
+    transform: scale(1);
+    }
+`;
+
+const pulseFade = keyframes`
+    0% {
+      opacity: 1;
+      transform: scale(1);
+    }
+    50% {
+      opacity: 0;
+      transform: scale(1.45);
+    }
+    100% {
+      opacity: 0;
+      transform: scale(3);
+    }
+`;
+
+export const Wrapper = styled.div`
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+`;
+
+export const LeftSide = styled.div`
+  ${({ theme }) => css`
+    background: ${theme.colors.mainBackground};
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 1;
+  `}
+`;
+
+export const RightSide = styled.div`
+  ${({ theme }) => css`
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex: 1;
+    background: ${theme.colors.primary};
+    align-items: center;
+    justify-content: center;
+  `}
+`;
+
+export const NoMatchLoader = styled(Logo)`
```

Comment:
Mówilem żeby skorzysyać z StyledLogo i StyledAltLogo. Tam są już animacje. DRY to taka zasada programowania = Don't repeat yourself. Korzystaj z tego co już jest a nie kopiuj :)

Reply:
Czyli poprawna procedura rozumiem powinna być taka?
1. Importujemy jakiś element do styled
2. Tworzymy nowy styled-component który dziedziczy po tym importowanym
3. Zmieniamy wartości na takie jakie są aktualnie potrzebne jednocześnie korzystając ze "starej" reszty